src/classes/BDI_DataImportCTRL_TEST.cls

```diff
@@ -221,6 +221,64 @@ private with sharing class BDI_DataImportCTRL_TEST {
         Test.stopTest();
     }
 
+    /*********************************************************************************************************
```

Comment:
@randi274 I added the test scenarios to cover the UseBatchGiftEntryGrid__c use cases. Can you do a quick check and see if I missed something. Then we can approve this PR.